### PR TITLE
feat(codelab): Add Azure to Kubernetes codelab

### PR DESCRIPTION
**Pull request by ejizba**

### CODE REVIEW

**Comment by ejizba** [comment]
@clguimanmsft @rguthriemsft @dcaro PTAL

**Review by lwander — APPROVED** [review]
Nice

**Comment by lwander** [comment]
Hey Eric - if you get the chance, could you create a codelab similar to lone Halyard quickstart here: https://spinnaker.github.io/setup/quickstart/halyard/, but for Azure + K8s? I think that should help stave off config questions early on. I'll be adding one for GCE + Halyard soon too.

File: guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Deploy Dev Ops VM
+
+Run the following command to deploy the 'Continuous Delivery to Kubernetes' [Quickstart Template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-acr-spinnaker-k8s) in Azure. It will make sure you have the [Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) installed and interactively ask if you want to change any of the default behavior. **Important:** When asked, make sure to specify your fork of the sample git repository so that you can trigger the pipeline with your own commits.
+
+```bash
+bash <(curl -sL https://aka.ms/DeployDevOps)
+```
+
+The script should list several outputs after it has finished. Run the 'SSH' output to connect to your Jenkins and Spinnaker instances hosted on the same VM.
```
Comment (clguiman) [inline guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md:24]: I'd add a sentence explaining what the SSH command does (what ports will be forwarded)

File: guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+> NOTE: The 'Destroy Previous Dev Cluster' stage is expected to have failed because there is no previous cluster yet
+
+After the pipeline finishes, go to the Cluster view to inspect your production environment. Unlike the 'dev' cluster, the 'prod' cluster will have a publicly available Ingress IP listed where you can verify your app. Click the icon for the load balancer under the 'prod' cluster as shown in the image below:
+
+![Verify Production IP](prod-ip.png)
+
+## Trigger Sample Pipeline
+
+Finally, make a change to your sample app in git and push your changes (for example, you can change the color or wording in the 'content/index.html' file). Follow the same steps above to watch as your new image is built and deployed to your staging/production environments.
```
Comment (clguiman) [inline guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md]: Maybe add a link to the Jenkins' GitHub plugin for further reading: https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin

**Review by clguiman — APPROVED** [review]

**Comment by ejizba** [comment]
@lwander Yep, can do!

Also I fixed the feedback and this PR is ready to be merged

**Pull request merged**